superagi/helper/knowledge_helper.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy.orm import Session
+import boto3
+import json
+from superagi.config.config import get_config
+from superagi.models.knowledge_config import KnowledgeConfig
+from superagi.models.index_config import VectorIndexConfig
+
+class KnowledgeHelper:
```

Comment:
helper classes does not make sense. Create a new package knowledge and restructure the classes.